shared/binlog.c

```diff
@@ -154,6 +154,12 @@ void binlog_wipe(binlog *bl, int flags)
 		unlink(bl->path);
 	}
 
+	if (bl->file_save_path) {
+		if( (flags == BINLOG_DELALL) && (access( bl->file_save_path, F_OK ) == 0) ) {
```

Comment:
The purpose of the flags variable seems to be a bit flag in ln147. 
However, here it is used as test for equality. Kindly conform to the existing code.

Reply:
Noted on this, will update this.